README.md

```diff
@@ -4,12 +4,12 @@
 - Use [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript)
 
 ## Tasks
-1. At first show a button `Start` on the page
-1. After clicking the button show a `GoodsList` (ul > li) with the given goods and hide the button
-1. Add `Reverse` button to show the goods in reversed order
-1. Add `Sort alphabetically` button to show the goods in alphabetical order
-1. Add `Reset` button to show the goods in the initial order
-1. Add `Sort by length` button to show the goods ordered by name length
+1. At first show a button with text `Start` on the page
+1. After clicking the button shows a `GoodsList` (ul > li) with the given goods and hide the `Start` button
```

Comment:
something is wrong with grammar here) 'after clicking the button, the list is shown ...' maybe? or if it's an instruction  - 'after clicking the button show ...'